app/views/incomeSources/add/ForeignPropertyCheckDetails.scala.html

```diff
@@ -93,12 +99,9 @@
             </div>
             <div class="govuk-summary-list__row">
                 <dt class="govuk-summary-list__key">@getMessage("accounting-method")</dt>
-                <dd class="govuk-summary-list__value" id="business-accounting-value">@{
-                    if (viewModel.cashOrAccrualsFlag == "cash") {
-                        getMessage("cash-basis-accounting")
-                    } else {
-                        getMessage("traditional-accounting")
-                    }
+                <dd class="govuk-summary-list__value" id="business-accounting-value">
+                    @{
+                    messages(accountingMethodMessageKey)
```

Comment:
Should fix: indentation here is off, this could probably be 1 line without curly braces now